ansible/roles/stack-sunbird/tasks/content-service.yml

```diff
@@ -30,6 +30,6 @@
   shell: "docker config create content-service_logback.xml /home/deployer/config/content-service_logback.xml"
 
 - name: Deploy content-service
-  shell: "docker service create --with-registry-auth --replicas {{ content_service_replicas }} -p 9002:9000  --name content-service --hostname content-service --reserve-memory {{ content_service_reservation_memory }} --limit-memory {{ content_service_limit_memory }} --limit-cpu {{ content_service_limit_cpu }} --reserve-cpu {{ content_service_reservation_cpu }} --health-cmd 'wget -qO- content-service:9000/health || exit 1' --health-timeout 3s --health-retries 3  --network application_default --config source=content-service.conf,target=/home/sunbird/content-service-1.0-SNAPSHOT/config/application.conf,mode=0644 --config source=content-service_logback.xml,target=/home/sunbird/content-service-1.0-SNAPSHOT/config/logback.xml,mode=0644  {{hub_org}}/{{image_name}}:{{image_tag}}"
+  shell: "docker service create --with-registry-auth --replicas {{ content_service_replicas }} -p 9002:9000  --name content-service --hostname content-service --reserve-memory {{ content_service_reservation_memory }} --limit-memory {{ content_service_limit_memory }} --limit-cpu {{ content_service_limit_cpu }} --reserve-cpu {{ content_service_reservation_cpu }} --health-cmd 'wget -qO- content-service:9000/health || exit 1' --health-timeout 10s --health-retries 3  --network application_default --config source=content-service.conf,target=/home/sunbird/content-service-1.0-SNAPSHOT/config/application.conf,mode=0644 --config source=content-service_logback.xml,target=/home/sunbird/content-service-1.0-SNAPSHOT/config/logback.xml,mode=0644  {{hub_org}}/{{image_name}}:{{image_tag}}"
```

Comment:
Change retry to 5 as we are trying 5 times in Kube.

Reply:
updated retry to 5 times